### DISCUSSION TITLE
The latest release containers a virus: Trojan:Win32/Vigorf.A

### DISCUSSION THREAD

**Discussion by joseche** [genotrance/px#194] [General]
The latest release contains a virus:
Trojan:Win32/Vigorf.A

**Comment by genotrance** [comment]
Can you please confimr how was this scanned and detected? File has been up for 8 months.

**Reply by genotrance** [reply]
Please see [here](https://github.com/genotrance/px/issues/182#issuecomment-1574292664) for what I'm considering as the solution. Feedback appreciated.

**Reply by genotrance** [reply]
[v0.9.0](https://github.com/genotrance/px/releases/tag/v0.9.0) has been posted - please review if this issue is resolved.
